darshan-util/pydarshan/pyproject.toml

```diff
@@ -3,3 +3,40 @@ requires = [
     "wheel",
     "setuptools",
 ]
+
+[tool.cibuildwheel]
+environment = "PYDARSHAN_BUILD_EXT=1"
+build = "pp39*"
+skip = [
+    "*musllinux*",
+    "*i686*"
+]
+test-requires = [
+    "pytest",
+    "lxml",
+    "matplotlib<3.5",
+    "importlib_resources;python_version<'3.9'"
+]
+test-command = "pytest {package}"
+
+[tool.cibuildwheel.linux]
+before-all = [
+    "yum install -y blas-devel lapack-devel",
+    "yum install -y libxslt-devel libxml2-devel",
+    "yum install -y libjpeg libjpeg-devel",
+    "git submodule update --init",
+    "./prepare.sh",
+    "./configure --disable-darshan-runtime --enable-apxc-mod --enable-apmpi-mod",
+    "make install"
+]
+
+[tool.cibuildwheel.macos]
```

Comment:
From the example: https://cibuildwheel.readthedocs.io/en/stable/options/#configuration-file
```
[tool.cibuildwheel.macos]
repair-wheel-command = [
  "delocate-listdeps {wheel}",
  "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} {wheel}",
]
```

And more info here:
https://cibuildwheel.readthedocs.io/en/stable/options/#repair-wheel-command